projects/client/RabbitMQ.Client/src/client/impl/Connection.cs

```diff
@@ -1170,13 +1176,11 @@ public override string ToString()
         public void WriteFrame(OutboundFrame f)
         {
             m_frameHandler.WriteFrame(f);
-            m_heartbeatWrite.Set();
         }
 
-        public void WriteFrameSet(IList<OutboundFrame> f)
+        public void WriteFrameSet(List<OutboundFrame> f)
```

Comment:
While this is better it is also a breaking change, was this intentional?

Reply:
It was but given how often IFrameHandler appears in (unnecessarily) public APIs we should probably revert it. :/